src/main.js

```diff
@@ -104,18 +109,41 @@ var currentPoster = {
 
 };
 
-currentPoster = new Poster('', '', '')
+//currentPoster = new Poster('', '', '')
 
 // event listeners go here 👇
+window.addEventListener('load', makeRandomPoster) {
```

Comment:
Self-note for Nina and Janika: Need to look at this event listener function and figure out what event will work. 'Load' does not seem to be working.